### ISSUE TITLE
NTR: Streptococcal infection in throat, by culture or rapid antigen (Lab: bacteriology, virology, parasitology, fungal)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8083]
New term request Streptococcal infection in throat, by culture or rapid antigen (hausma_130829202037): Rapid strep; group A beta-hemolytic strep  UMLS: 0241384

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000733
- label: Positive streptococcus A rapid antigen detection test
- parents: [Positive oropharangeal infectious agent test (HP:6000579)](https://hpo.jax.org/app/browse/term/HP:6000579)
- pmid: PMID:36941540
- def: Positive detection of a group A streptococcal antigen in a test perforned on a throat swab.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8083
- synonyms:
